actors/evm/src/interpreter/precompiles/fvm.rs

```diff
@@ -123,11 +123,12 @@ pub(super) fn lookup_delegated_address<RT: Runtime>(
     let id = id_bytes.next_param_padded::<u64>()?;
 
     let address = system.rt.lookup_delegated_address(id);
-    let ab = match address {
+    log::debug!("lookup_delegated_address: {:?}", address);
```

Comment:
For syscalls, we should just log in the mock runtime, and in the FVM itself (IIRC, we already do). Otherwise, we're just paying the syscall cost twice for something we already know in the FVM.

(yeah, this won't cost much when debugging is disabled, but it's still unnecessary)